content-src/actions/action-manager.js

```diff
@@ -82,7 +89,7 @@ function RequestRecentLinks(options) {
 }
 
 function RequestMoreRecentLinks(beforeDate) {
-  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE", {
+  return RequestRecentLinks({
```

Comment:
nit: this change isn't concerned with bookmarks and should probably be in a separate commit